src/components/generate/PlaylistGetUrlLink/PlaylistGetUrlLink.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button } from "@components/ui/Button"
```

Comment:
On playlist url copy, I am missing some effect to acknowledge the copy operation (dialog, pop-up, etc). Could we add that?

Reply:
This effect has been implemented in this task [FW-55](https://festwrap.atlassian.net/browse/FW-55)